File: Signe_Josue/exercise_13.cpp
#include <iostream>


/**
 * max - Find the maximum number in an array.
 * @arr: An array of integers.
 * @size: number of elements in the @arr.
 * 
 * Return: The maximum number in @arr.
*/
int maxNumber(int arr[], int size)
{
    int max = arr[0];
    for (int i = 1; i < size; i++)
    {
        if (arr[i] > max)
            max = arr[i];
    }
    return max;
}


/**
 * main- Entry point into the program.
 * 
 * Returns: Always 0.
 */
int main()
{
    int my_array[7] = {1, 5, 7, -2, -100, 34, 26};
    int result = maxNumber(my_array, 7);

    std::cout << result << std::endl;
    return 0;
}

File: Emy/exercise_13.php
<?php
//finds maximum number in an array
function getMax($array){
    for($i=0;$i<count($array);$i++){
        if($array[$i]>$array[$i+1]){
            $t = $array[$i];
            $array[$i] = $array[$i+1];
            $array[$i+1] = $t;
        }
    }
    $max = count($array)-1;
    return $max;
}

$arr= [6, 4, 2, 0, 9];
$result = getMax($arr);
var_dump($result);